### PR TITLE
fix the details changer format for return

### PR DESCRIPTION
**Pull request by zachcd**
return the changed user on edit

### CODE REVIEW

**Review by HTJin — APPROVED** [review]

File: src/Details/index.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const detailsChanger = (stream, user, id, body) => {
   console.log(body)
```
Comment (HTJin) [inline src/Details/index.js]: semicolon

File: src/Draft/index.js [hunk 2]
```diff
@@ -32,19 +31,19 @@ const draftHandler = (stream, body, user) => {
...
           currentDraft = {started: false};
         });
-
         // waiting on a confirmation event
         currentDraft.events.on('waitingConfirmation', (event) => {
           console.log('DRAFT', ' : ', 'waiting on a confirmation');
           draftSub.fire('waitingConfirmation', JSON.stringify(event));
         });
+
+
```
Comment (HTJin) [inline src/Draft/index.js:46]: random empty lines?

Comment (zachcd) [reply src/Draft/index.js:46]: intentional

File: src/Details/index.js (hunk 1 above)
Comment (zachcd) [reply src/Details/index.js]: not in this commit :P

**Pull request merged**